src/main/java/org/stellar/sdk/SorobanServer.java

```diff
@@ -5,12 +5,7 @@
 import java.io.Closeable;
 import java.io.IOException;
 import java.net.SocketTimeoutException;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
-import java.util.Optional;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Please avoid using wildcard imports. I know both have their advantages, but this has been discussed in other PRs before. Since we have a standard now, let's adhere to it.
https://github.com/lightsail-network/java-stellar-sdk/blob/master/CONTRIBUTING.md#code-style

Reply:
This is actually from the auto formatter, which is google-java-formatter. What do you think we let the formatter handle this?

Reply:
I don't think google-java-formatter does this; are you using IDEA? IDEA does this by default.
https://www.baeldung.com/intellij-disable-wildcard-import

Reply:
Sorry, I just tested it, and it was really modified by google-java-formatter.